eras/conway/impl/src/Cardano/Ledger/Conway/Rules/Epoch.hs

```diff
@@ -238,12 +235,15 @@ epochTransition = do
       epochState' =
         EpochState
           acnt'
-          (returnProposalDeposits adjustedLState)
+          adjustedLState
           ss'
           nm
 
   let
-    utxoSt''' = utxoSt' {utxosDeposited = obligationCertState adjustedCertState}
+    utxoSt''' =
+      utxoSt'
+        { utxosDeposited = totalObligation adjustedCertState utxoSt'
+        }
     acnt'' = acnt' {asReserves = asReserves acnt'}
```

Comment:
I doubt whether this statement does anything meaningful and is a no-op.

Reply:
There are some changes in deposits after running the `POOLREAP` rule I think.

Reply:
oh, right, I looked at the lines before it, you're right about `acnt''`